### ISSUE TITLE
Optional Supression of Date Parse Warning?

### ISSUE THREAD

**Issue by sdmcallister** [bluenote10/NimData#54]
Does this seem like a sensible option to add? 

I have a date column where there are tens of thousands of 0 values.  I like the warning intially, but very quickly it fills up the terminal. 

To avoid the warnings I could (should I?) try to handle this  prior to passing the data to the schema parser using a regex?  Maybe just use an intCol and create a new column that is parsed? Any suggestions are appreciated.

**Comment by bluenote10** [comment]
Yes this has been [marked as a todo](https://github.com/bluenote10/NimData/blob/c29312618f826598b40ff009f6583794e066559b/src/nimdata/schema_parser.nim#L234-L244) for a while, but I never had a good idea how aggregate these errors in a way that doesn't impact complexity too much. Probably we would have to use `Option[Time]` in general then?

In practice you probably can work-around the problem by keeping the field as a string initially, and then apply a mapper with your own string-to-date conversion with fallbacks.
